tests/targets/test_ios.py

```diff
@@ -78,3 +89,131 @@ def test_check_configuration_tokens(self):
                 ]
             )
         ]
+
+    def test_get_available_packages(self):
+        """Checks the toolchain `recipes --compact` output is parsed correctly to return recipe list."""
+        target = init_target(self.temp_dir)
+        with patch_target_ios("toolchain") as m_toolchain:
+            m_toolchain.return_value = ("hostpython3 kivy pillow python3 sdl2", None, 0)
+            available_packages = target.get_available_packages()
+        assert m_toolchain.call_args_list == [
+            mock.call("recipes --compact", get_stdout=True)
+        ]
+        assert available_packages == [
+            "hostpython3",
+            "kivy",
+            "pillow",
+            "python3",
+            "sdl2",
+        ]
+
+    def test_install_platform(self):
+        """Checks `install_platform()` calls clone commands and sets `ios_dir` and `ios_deploy_dir` attributes."""
+        target = init_target(self.temp_dir)
+        assert target.ios_dir is None
+        assert target.ios_deploy_dir is None
+        with patch_buildozer_cmd() as m_cmd:
+            target.install_platform()
+        assert m_cmd.call_args_list == [
+            mock.call("git clone https://github.com/kivy/kivy-ios", cwd=mock.ANY),
+            mock.call(
+                "git clone --branch 1.10.0 https://github.com/phonegap/ios-deploy",
+                cwd=mock.ANY,
+            ),
+        ]
+        assert target.ios_dir.endswith(".buildozer/ios/platform/kivy-ios")
+        assert target.ios_deploy_dir.endswith(".buildozer/ios/platform/ios-deploy")
+
+    def test_compile_platform(self):
+        """Checks the `toolchain build` command is called on the ios requirements."""
+        target = init_target(self.temp_dir)
+        target.ios_deploy_dir = "/ios/deploy/dir"
+        # fmt: off
```

Comment:
Thanks for sharing.
Yes exactly it's to avoid black from formatting it (they're currently migrating [from parenthesis to backslash](https://github.com/psf/black/blob/7403d95/docs/the_black_code_style.md#how-black-wraps-lines) after being convinced).
I personally don't care because I'm a pragmatic so I tend to just leave things the default. However I received a comment here (https://github.com/kivy/buildozer/pull/1168#discussion_r446519581) that the parenthesis were disliked.
I think this is all just a matter of taste and opinion which is exactly the goal of black, to decide for us so we don't waste too much time talking about style and just chose a linter that does the job.
Hence until other devs are aligned together I'll leave it this way if you don't mind